app/models/article.rb

```diff
@@ -416,6 +416,22 @@ def access_by?(user)
     user.admin? || user_id == user.id
   end
 
+  def merge_with(other_article_id)
+    if !Article.exists?(other_article_id) || other_article_id == self.id
+      return self
+    else
+      other_article = Article.find(other_article_id)
+      self.body = self.body + "\n\n" + other_article.body
+      other_article.comments.each do |comment|
+        self.comments << comment
```

Comment:
It's possible that the `<<` operator will work list to list - it might also be good to choose something that concats the lists instead of needing to iterate